### PR TITLE
Adds configurations and jobs for Multus-Networkpolicy

### PR DESCRIPTION
**Pull request by dougbtv**
Related ART request: [ART-2286](https://issues.redhat.com/browse/ART-2286)

### CODE REVIEW

File: ci-operator/config/openshift-priv/multus-networkpolicy/openshift-priv-multus-networkpolicy-cni-release-4.8.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
+base_images:
```
Comment (petr-muller) [inline ci-operator/config/openshift-priv/multus-networkpolicy/openshift-priv-multus-networkpolicy-cni-release-4.8.yaml:1]: PLease remove all files and config for `openshift-priv/multus-networkpolicy` and keep just the ones for `openshift/multus`. The config for `openshift-priv` is completely inferred from the one for the public repo (see https://docs.ci.openshift.org/docs/architecture/private-repositories/#openshift-priv-organization for more information)

**Review by petr-muller — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
+base_images:
```
Comment (petr-muller) [inline ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml:1]: Also remove config for any release-4.7+ release branches - future-looking branches config is also automatically managed.

**Review by petr-muller — CHANGES_REQUESTED** [review]

File: ci-operator/config/openshift-priv/multus-networkpolicy/openshift-priv-multus-networkpolicy-cni-release-4.8.yaml (hunk 1 above)
Comment (dougbtv) [reply ci-operator/config/openshift-priv/multus-networkpolicy/openshift-priv-multus-networkpolicy-cni-release-4.8.yaml]: Petr, thanks for the eyes here! I wasn't aware of that. appreciate, got those removed.

File: ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml (hunk 2 above)
Comment (dougbtv) [reply ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml:1]: Excellent, the 4.8 release files have also been removed :+1:

Comment (petr-muller) [reply ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml:1]: They still seem to be here though ;)

File: ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-master.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    namespace: ocp
+    tag: rhel-7-golang-openshift-4.6
+  ocp_builder_rhel-8-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-1.15-openshift-4.6
+  ocp_builder_rhel-8-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
```
Comment (petr-muller) [inline ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-master.yaml:21]: This looks unused

File: ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-master.yaml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    namespace: ocp
+    tag: base
+  ocp_builder_rhel-7-golang-1.15-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-7-golang-1.15-openshift-4.6
+  ocp_builder_rhel-7-golang-openshift-4.6:
+    name: builder
+    namespace: ocp
+    tag: rhel-7-golang-openshift-4.6
```
Comment (petr-muller) [inline ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-master.yaml:13]: This looks unused

File: ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml (hunk 2 above)
Comment (dougbtv) [reply ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml:1]: 4.7+! Ah ha! Shoot thanks for the help Petr, I just took out the 4.8 files, thanks for that.

File: ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-master.yaml (hunk 3 above)
Comment (dougbtv) [reply ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-master.yaml]: Removed, thanks!

File: ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-master.yaml (hunk 4 above)
Comment (dougbtv) [reply ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-master.yaml]: Got it, thanks again.

File: ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml (hunk 2 above)
Comment (petr-muller) [reply ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml:1]: One more deletion please :)

`ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml`

**Comment by petr-muller** [comment]
The generators changed in the meantime, too - it's safest to rebase and run `make update` to pick up all changes.

**Comment by dougbtv** [comment]
Rebased and ran `make update`

**Comment by dougbtv** [comment]
ah ha, new commit up where I delete `ci-operator/config/openshift/multus-networkpolicy/openshift-multus-networkpolicy-release-4.7.yaml` (missed that)

**Review by petr-muller — APPROVED** [review]

**Comment by petr-muller** [comment]
:+1: thx for patience

**Pull request merged**